plugins/woocommerce-admin/client/marketplace/marketplace.scss

```diff
@@ -3,17 +3,17 @@
 .woocommerce-admin-page__extensions {
 	background: #fff;
 
+	#wpbody {
+		margin: 0;
+	}
+
 	#wpbody-content {
 		/* Prevent double-scrollbar issue on WooCommerce > Extension pages */
 		overflow: hidden;
 	}
 
 	.woocommerce-layout__primary {
-		margin: $header-height-mobile 0 0;
-
-		@media (min-width: $breakpoint-medium) {
-			margin-top: $header-height-desktop;
-		}
```

Comment:
We can't hardcode the header height now. It changes depending on the tab. Since we're not doing `top: -60px` anymore we can stop hardcoding it and let it flow by itself.